src/components/ActionBar.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="vxp-action-bar">
+    <span v-if="title && showTitle" class="vxp-action-bar__title">{{ title }}</span>
```

Comment:
could you please check if linter did process this file?
you can run
`npm run lint`